ardcvocabs/src/main/java/au/org/aodn/ardcvocabs/model/VocabApiPaths.java

```diff
@@ -1,37 +1,36 @@
 package au.org.aodn.ardcvocabs.model;
-
 import lombok.Getter;
 
 @Getter
 public enum VocabApiPaths {
     PARAMETER_VOCAB(
-            "/aodn-parameter-category-vocabulary/version-2-1/concept.json",
-            "/aodn-parameter-category-vocabulary/version-2-1/resource.json?uri=%s",
-            "/aodn-discovery-parameter-vocabulary/version-1-6/concept.json",
-            "/aodn-discovery-parameter-vocabulary/version-1-6/resource.json?uri=%s"
+            "/aodn-parameter-category-vocabulary/%s/concept.json",
+            "/aodn-parameter-category-vocabulary/%s/resource.json?uri=%s",
+            "/aodn-discovery-parameter-vocabulary/%s/concept.json",
+            "/aodn-discovery-parameter-vocabulary/%s/resource.json?uri=%s"
```

Comment:
although I could replace `version` directly with `current`, writing tests is a bit headache, after debugging I found that links like this for example:

`https://vocabs.ardc.edu.au/repository/api/lda/aodn/aodn-platform-vocabulary/current/concept.json`
or 
`https://vocabs.ardc.edu.au/repository/api/lda/aodn/aodn-platform-vocabulary/current/concept.json?_page=1`

the `next` and `prev` fields refer to paths with specific `version` instead of `current`.

So less complex tests and later on ... check if there are versions change to update caches, I keep this template enum, and have separate method for extracting latest specific versions from `current`, `current` is just a starting point wrapper.